pkg/assembler/backends/testing/cve.go

```diff
@@ -17,69 +17,199 @@ package testing
 
 import (
 	"context"
+	"fmt"
+	"log"
+	"strconv"
 	"strings"
 
 	"github.com/guacsec/guac/pkg/assembler/graphql/model"
+	"github.com/vektah/gqlparser/v2/gqlerror"
 )
 
 func registerAllCVE(client *demoClient) {
-	client.registerCVE("2019", "CVE-2019-13110")
-	client.registerCVE("2014", "CVE-2014-8139")
-	client.registerCVE("2014", "CVE-2014-8140")
-	client.registerCVE("2022", "CVE-2022-26499")
-	client.registerCVE("2014", "CVE-2014-8140")
-}
-
-// Ingest CVE
+	ctx := context.Background()
 
-func (c *demoClient) registerCVE(year, id string) *model.Cve {
-	idLower := strings.ToLower(id)
-	for i, s := range c.cve {
-		if s.Year == year {
-			c.cve[i] = registerCveID(s, idLower)
-			return c.cve[i]
+	inputs := []model.CVEInputSpec{{
+		Year:  "2019",
+		CveID: "CVE-2019-13110",
+	}, {
+		Year:  "2014",
+		CveID: "CVE-2014-8139",
+	}, {
+		Year:  "2014",
+		CveID: "CVE-2014-8140",
+	}, {
+		Year:  "2022",
+		CveID: "CVE-2022-26499",
+	}, {
+		Year:  "2014",
+		CveID: "CVE-2014-8140",
+	}}
+	for _, input := range inputs {
+		_, err := client.IngestCve(ctx, &input)
+		if err != nil {
+			log.Printf("Error in ingesting: %v\n", err)
 		}
 	}
+}
 
-	newCve := &model.Cve{Year: year}
-	newCve = registerCveID(newCve, idLower)
-	c.cve = append(c.cve, newCve)
-
-	return newCve
+// Internal data: osv
+type cveMap map[string]*cveNode
+type cveNode struct {
+	id     uint32
+	year   string
```

Comment:
Should the year be an int?

Reply:
Changing it to Int for the year in the schema.

Reply:
I will make another PR to make this change as this one is already large.